### ISSUE TITLE
FXCanvas does not update x/y of EmbeddedStageInterface when FXCanvas is reparented

### ISSUE THREAD

**Issue by tomsontom** [javafxports/openjdk-jfx#138]
This is a clone of https://bugs.openjdk.java.net/browse/JDK-8207794

**Comment by tomsontom** [comment]
@kevinrushforth would you have time to review that

**Comment by kevinrushforth** [comment]
OK, I can review it early next week. Can you send an email to the openjfx-dev list with a pointer to the review so others can see it if they are interested (and so it can be merged into the openjdk jfx-dev repo once this review is done)?
